frontend/packages/console-app/src/components/nav/FavoriteNavItemResources.scss

```diff
@@ -0,0 +1,38 @@
+.oc-favorite-resource.pf-v6-c-nav__item {
+  .pf-v6-c-nav__link {
+    display: flex;
```

Comment:
Could reduce the amount of declarations here by using https://www.patternfly.org/layouts/flex to achieve the desired layout.